spec/requests/admin/configs_spec.rb

```diff
@@ -194,74 +194,96 @@
 
       describe "Community Content" do
         it "updates the community_description" do
-          allow(SiteConfig).to receive(:community_description).and_call_original
+          allow(Settings::Community).to receive(:community_description).and_call_original
           description = "Hey hey #{rand(100)}"
-          post "/admin/config", params: { site_config: { community_description: description },
-                                          confirmation: confirmation_message }
-          expect(SiteConfig.community_description).to eq(description)
+          post admin_settings_communities_path, params: {
+            settings_community: { community_description: description },
+            confirmation: confirmation_message
+          }
+          expect(Settings::Community.community_description).to eq(description)
         end
 
         it "updates the community_emoji if valid" do
-          allow(SiteConfig).to receive(:community_emoji).and_call_original
+          allow(Settings::Community).to receive(:community_emoji).and_call_original
           emoji = "🥐"
-          post "/admin/config", params: { site_config: { community_emoji: emoji },
-                                          confirmation: confirmation_message }
-          expect(SiteConfig.community_emoji).to eq(emoji)
+          post admin_settings_communities_path, params: {
+            settings_community: { community_emoji: emoji },
+            confirmation: confirmation_message
+          }
+          expect(Settings::Community.community_emoji).to eq(emoji)
         end
 
         it "does not update the community_emoji if invalid" do
-          allow(SiteConfig).to receive(:community_emoji).and_call_original
+          Settings::Community.community_emoji = "🥐"
```

Comment:
I'm honestly not quite sure why I need to do this but without explicitly setting the emoji this spec would only be green in isolation, not when running the whole file or suite.